src/main/java/recipeditor/command/ExitCommand.java

```diff
@@ -1,20 +1,15 @@
 package recipeditor.command;
 
-import recipeditor.recipe.RecipeList;
-import recipeditor.recipe.Recipe;
 import recipeditor.storage.Storage;
 
-import java.io.File;
-
-
 public class ExitCommand extends Command {
     public static final String COMMAND_TYPE = "/exit";
     public static final String EXIT_MESSAGE = "RecipEditor ends here...";
-    private static final String COMMAND_SYNTAX = "/exit";
+    public static final String COMMAND_NAME = "exit";
     private static final String COMMAND_FUNCTION = "Exit recipeditor.";
 
     public ExitCommand() {
-        super(COMMAND_SYNTAX, COMMAND_FUNCTION);
+        super(COMMAND_TYPE, COMMAND_FUNCTION);
```

Comment:
similar to in `AddCommand`, `COMMAND_SYNTAX` and `COMMAND_TYPE` are same value but different meaning